charts/gateway-addons-helm/Chart.yaml

```diff
@@ -26,8 +26,20 @@ sources:
 
 dependencies:
   - name: prometheus
-    version: 23.1.0
+    version: 25.21.0
     repository: https://prometheus-community.github.io/helm-charts
   - name: grafana
     repository: https://grafana.github.io/helm-charts
-    version: 6.58.4
+    version: 8.0.0
+  - name: fluent-bit
```

Comment:
IMO, should add `condition` which give user ability to disable subchart.

Reply:
good point!